lib/screens_web/views/v2/audio/departures_view.ex

```diff
@@ -64,8 +64,14 @@ defmodule ScreensWeb.V2.Audio.DeparturesView do
     ~E|<s><%= content %></s>|
   end
 
-  defp render_time_with_crowding({%{crowding: crowding, time: time}, _}, route, _headsign) do
-    route_headsign_rendered = render_route_headsign(route, nil)
+  defp render_time_with_crowding({%{crowding: crowding, time: time}, _}, route, headsign) do
+    route_headsign_rendered =
+      if String.starts_with?(route.route_text, "Green Line") do
```

Comment:
Any way we can flag this special case at a level where we still have access to the route ID? Making the behavior conditional on the user-visible (or rather user-audible) name of the route feels fragile.